packages/pharos-site/static/guidelines/toggle-button-group.docs.mdx

```diff
@@ -0,0 +1,143 @@
+        <li>Don't use an icon-only toggle if the actions aren't clearly understood</li>
+        <li>Don't use the button group component if there are more than three options</li>
+        <li>Avoid wrapping the toggle button group onto more than a single row.</li>
+      </ul>
+    }
+  />
+</PageSection>
+<PageSection title="Content guidelines">
+  <p>If using labels, they should be concise and descriptive, using no more than three words.</p>
+  <PageSection subSectionLevel={1} title="States">
+    <PharosHeading level="4" preset="1--bold">
+      Default
+    </PharosHeading>
+    <p>Indicates that the user can interact with the button.</p>
+    <PharosHeading level="4" preset="1--bold">
+      Selected
+    </PharosHeading>
+    <p>Indicates that the button is selected.</p>
+    <PharosHeading level="4" preset="1--bold">
+      Disabled
+    </PharosHeading>
+    <p>Indicates that the user is unable to interact with the button.</p>
+  </PageSection>
+</PageSection>
+<PageSection title="Variants">
+  <PageSection title="Icon + Text" subSectionLevel={1}>
+    <PharosToggleButtonGroup>
+      <PharosToggleButton id="gallery-mode-button" icon-left="view-gallery">
+        Gallery
+      </PharosToggleButton>
+      <PharosToggleButton id="list-mode-button" icon-left="view-list">
+        List
+      </PharosToggleButton>
+    </PharosToggleButtonGroup>
+  </PageSection>
+  <PageSection title="Text only" subSectionLevel={1}>
+    <PharosToggleButtonGroup>
+      <PharosToggleButton id="gallery-mode-button">Gallery</PharosToggleButton>
+      <PharosToggleButton id="list-mode-button">List</PharosToggleButton>
+    </PharosToggleButtonGroup>
+  </PageSection>
+  <PageSection title="Icon only" subSectionLevel={1}>
+    <PharosToggleButtonGroup>
+      <PharosToggleButton id="gallery-mode-button" icon-left="view-gallery" />
+      <PharosToggleButton id="list-mode-button" icon-left="view-list" />
+    </PharosToggleButtonGroup>
+  </PageSection>
+</PageSection>
+
+<PageSection title="Accessibility">
+  <PageSection subSectionLevel={1} title="Relevant WCAG guidelines">
+    <ul>
+      <li>
+        <PharosLink href="https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html">
+          1.3.1 Info and Relationships A
+        </PharosLink>
+      </li>
+    </ul>
+  </PageSection>
+  <PageSection title="Code expectations" subSectionLevel={1}>
+    <ul>
+      <li>
+        There is a <code>role="group"</code> tag on the parent element <code>&lt;div&gt;</code>
+        <ul>
+          <li>
+            If the toggle button group is part of a toolbar use <code>role="toolbar"</code>
+          </li>
+        </ul>
+      </li>
+      <li>
+        Use the semantic <code> &lt;button&gt; </code> on all grouped elements under the parent
+        element
+      </li>
+      <li>
+        If the button needs additional information in order to be understood by assistive
+        technologies, use an aria-label to convey this information
```

Comment:
Either "use an `aria-label` attribute" or "use an ARIA label" might be better here.